service/src/main/java/bio/terra/workspace/service/workspace/flight/DeleteAzureContextFlight.java

```diff
@@ -0,0 +1,26 @@
+package bio.terra.workspace.service.workspace.flight;
+
+import bio.terra.stairway.Flight;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.RetryRule;
+import bio.terra.workspace.common.utils.FlightBeanBag;
+import bio.terra.workspace.common.utils.RetryRules;
+import java.util.UUID;
+
+public class DeleteAzureContextFlight extends Flight {
+  public DeleteAzureContextFlight(FlightMap inputParameters, Object applicationContext) {
+    super(inputParameters, applicationContext);
+
+    FlightBeanBag appContext = FlightBeanBag.getFromObject(applicationContext);
+    UUID workspaceId =
+        UUID.fromString(inputParameters.get(WorkspaceFlightMapKeys.WORKSPACE_ID, String.class));
+
+    RetryRule retryRule = RetryRules.cloudLongRunning();
+
+    // TODO cleanup any azure resources here
```

Comment:
Can we create a ticket for this or will it be done as part of a different ticket?

Reply:
@marctalbott good idea, [done](https://broadworkbench.atlassian.net/browse/WOR-139)